crux_time/src/lib.rs

```diff
@@ -12,8 +12,11 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 pub struct TimeRequest;
 
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct TimeResponse(pub String);
+
 impl Operation for TimeRequest {
-    type Output = String;
+    type Output = TimeResponse;
```

Comment:
Interesting! What forced that?

Reply:
Yeah, was struggling without a wrapper. The typegen creates types we can call serialize and deserialize on